app/assets/javascripts/discourse/ember/resolver.js.es6

```diff
@@ -188,7 +188,8 @@ export default Ember.DefaultResolver.extend({
       const dashed = decamelized.replace(/_/g, '-');
       return Ember.TEMPLATES[decamelized] ||
              Ember.TEMPLATES[dashed] ||
-             Ember.TEMPLATES[dashed.replace('admin-', 'admin/')];
+             Ember.TEMPLATES[dashed.replace('admin-', 'admin/')] ||
+             Ember.TEMPLATES[dashed.replace('admin-', 'admin/templates/')];
```

Comment:
I was attempting to add an admin template to a plugin, and found that the resolver didn't pick it up because 'decamelized' here was `javascripts/admin_chats`, which means it was looking for `javascripts/admin/chats`, when the template is located at `javascripts/admin/templates/chats`

I left the existing `dashed.replace('admin-')` bit because I wasn't sure if that was a mistake or if other routes use that correctly.